src/scss/variables.scss

```diff
@@ -16,7 +16,7 @@ $grey-light: #d6cfde;
 
 $id-color: #127db3;
 $link-color: #6c2db5;
-$title-color: #2f0d56;
+$title-color: #300E56;
```

Comment:
`title-color` has a wide range of effects. `#2f0d56` is used in many places.
Has the designer been able to confirm this change?

Reply:
i will check it